Managers/UInputMan.cpp

```diff
@@ -840,20 +840,25 @@ namespace RTE {
 		}
 
 		if (g_InActivity) {	
-			bool BuyMenuIsOpen = 0;
-			GameActivity* temp = dynamic_cast<GameActivity*>(g_ActivityMan.GetActivity());
 
-			// Make ESC key open BuyMenuGUIs if any are open rather than pausing the game.
-			for (short player = Players::PlayerOne; player < Players::MaxPlayerCount; player++) {
-				if (temp->GetBuyGUI(player)->IsVisible()) {
-					BuyMenuIsOpen = true;
-					break;
+			if (AnyStartPress(false)) {
+
```

Comment:
A few minor comments
1. Probably get rid of this whitespace here on line 845. We usually don't have whitespace between if conditions and the first line of logic, except maybe if the if condition or logic line are excessively long.
2. Initialize `buyMenuIsOpen` with `false` instead of `0` please. The result is the same but it's just a little bit more sensible.
3. Remove the comment on line 849. We generally try to avoid comments unless they explain why we're doing something, or some very dense, complicated code, and I wouldn't say this needs it.